liquibase-standard/src/main/java/liquibase/executor/ExecutorService.java

```diff
@@ -134,7 +134,9 @@ private static class Key {
         }
 
         private String normalizeExecutorName(String executorName) {
-            return executorName.toLowerCase();
+            if (executorName == null) { return ""; } else {
```

Comment:
I  believe with the other fixes , this change is not required anymore. Could you rollback it?

Reply:
done